recipes/saxpy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "saxpy" %}
+{% set version = "1.0.1.dev167" %}
```

Comment:
This is a dev release and should accompany a conda-forge yaml file to send it to a dev channel. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Would something like this work? Just repushed. This is a package I need as a dependency for another package I'm trying to get on conda forge, would packages on a dev channel be available?

Reply:
I think you would need to add the dev channel with the conda-forge.yml in the other package's feedstock. https://conda-forge.org/docs/maintainer/conda_forge_yml.html#channels